crates/mysten-common/src/logging.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+#[macro_export]
+macro_rules! fatal {
+    ($($arg:tt)*) => {{
+        tracing::error!($($arg)*);
+        panic!($($arg)*);
```

Comment:
There is no guarantee that the log must be right before the panic, right?

Reply:
Correct, but they should be relatively close together. And easier to find.

I added a field to error message in this case to make it more easily findable in loki.